### PR TITLE
fixed RiC / eoTV & changed tvg-ids for eoTV HD, Bibel TV Impuls, Bibel TV Musik, Radio Bremen TV HD

### PR DESCRIPTION
**Pull request by freddii**

### CODE REVIEW

**Review by freddii — COMMENTED** [review]
fixed https://github.com/jnk22/kodinerds-iptv/issues/332

**Review by freddii — COMMENTED** [review]
fixed https://github.com/jnk22/kodinerds-iptv/issues/318

**Review by freddii — COMMENTED** [review]
fixed https://github.com/jnk22/kodinerds-iptv/issues/324

**Review by freddii — COMMENTED** [review]
fixed https://github.com/jnk22/kodinerds-iptv/issues/331

**Review by freddii — COMMENTED** [review]
fixed https://github.com/jnk22/kodinerds-iptv/issues/287

File: iptv/clean/clean.m3u [hunk 1]
```diff
@@ -6,7 +6,7 @@ https://zdf-hls-15.akamaized.net/hls/live/2016498/de/veryhigh/master.m3u8
 #EXTINF:-1 tvg-name="3sat" tvg-id="3sat.de" group-title="IPTV-DE" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/3sat.png",3sat
 https://zdf-hls-18.akamaized.net/hls/live/2016501/dach/veryhigh/master.m3u8
 #EXTINF:-1 tvg-name="ARTE HD" tvg-id="ARTE.de" group-title="IPTV-DE" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/artehd.png",ARTE HD
-https://artelive-lh.akamaihd.net/i/artelive_de@393591/index_1_av-b.m3u8
+https://artelive-lh.akamaihd.net/i/artelive_de@393591/master.m3u8
```
Comment (jnk22) [inline iptv/clean/clean.m3u:9]:
```suggestion
https://artelive-lh.akamaihd.net/i/artelive_de@393591/index_1_av-p.m3u8
```

File: iptv/clean/clean_tv_main.m3u [hunk 2]
```diff
@@ -6,7 +6,7 @@ https://zdf-hls-15.akamaized.net/hls/live/2016498/de/veryhigh/master.m3u8
 #EXTINF:-1 tvg-name="3sat" tvg-id="3sat.de" group-title="IPTV-DE" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/3sat.png",3sat
 https://zdf-hls-18.akamaized.net/hls/live/2016501/dach/veryhigh/master.m3u8
 #EXTINF:-1 tvg-name="ARTE HD" tvg-id="ARTE.de" group-title="IPTV-DE" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/artehd.png",ARTE HD
-https://artelive-lh.akamaihd.net/i/artelive_de@393591/index_1_av-b.m3u8
+https://artelive-lh.akamaihd.net/i/artelive_de@393591/master.m3u8
```
Comment (jnk22) [inline iptv/clean/clean_tv_main.m3u:9]:
```suggestion
https://artelive-lh.akamaihd.net/i/artelive_de@393591/index_1_av-p.m3u8
```

File: iptv/clean/clean_tv.m3u [hunk 3]
```diff
@@ -6,7 +6,7 @@ https://zdf-hls-15.akamaized.net/hls/live/2016498/de/veryhigh/master.m3u8
 #EXTINF:-1 tvg-name="3sat" tvg-id="3sat.de" group-title="IPTV-DE" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/3sat.png",3sat
 https://zdf-hls-18.akamaized.net/hls/live/2016501/dach/veryhigh/master.m3u8
 #EXTINF:-1 tvg-name="ARTE HD" tvg-id="ARTE.de" group-title="IPTV-DE" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/artehd.png",ARTE HD
-https://artelive-lh.akamaihd.net/i/artelive_de@393591/index_1_av-b.m3u8
+https://artelive-lh.akamaihd.net/i/artelive_de@393591/master.m3u8
```
Comment (jnk22) [inline iptv/clean/clean_tv.m3u:9]:
```suggestion
https://artelive-lh.akamaihd.net/i/artelive_de@393591/index_1_av-p.m3u8
```

File: iptv/kodi/kodi_tv_main.m3u [hunk 4]
```diff
@@ -6,7 +6,7 @@ https://zdf-hls-15.akamaized.net/hls/live/2016498/de/veryhigh/master.m3u8
 #EXTINF:-1 tvg-name="3sat" tvg-id="3sat.de" group-title="Vollprogramm" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/3sat.png",3sat
 https://zdf-hls-18.akamaized.net/hls/live/2016501/dach/veryhigh/master.m3u8
 #EXTINF:-1 tvg-name="ARTE HD" tvg-id="ARTE.de" group-title="Vollprogramm" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/artehd.png",ARTE HD
-https://artelive-lh.akamaihd.net/i/artelive_de@393591/index_1_av-b.m3u8
+https://artelive-lh.akamaihd.net/i/artelive_de@393591/master.m3u8
```
Comment (jnk22) [inline iptv/kodi/kodi_tv_main.m3u:9]:
```suggestion
https://artelive-lh.akamaihd.net/i/artelive_de@393591/index_1_av-p.m3u8
```

File: iptv/kodi/kodi_tv.m3u [hunk 5]
```diff
@@ -6,7 +6,7 @@ https://zdf-hls-15.akamaized.net/hls/live/2016498/de/veryhigh/master.m3u8
 #EXTINF:-1 tvg-name="3sat" tvg-id="3sat.de" group-title="Vollprogramm" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/3sat.png",3sat
 https://zdf-hls-18.akamaized.net/hls/live/2016501/dach/veryhigh/master.m3u8
 #EXTINF:-1 tvg-name="ARTE HD" tvg-id="ARTE.de" group-title="Vollprogramm" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/artehd.png",ARTE HD
-https://artelive-lh.akamaihd.net/i/artelive_de@393591/index_1_av-b.m3u8
+https://artelive-lh.akamaihd.net/i/artelive_de@393591/master.m3u8
```
Comment (jnk22) [inline iptv/kodi/kodi_tv.m3u:9]:
```suggestion
https://artelive-lh.akamaihd.net/i/artelive_de@393591/index_1_av-p.m3u8
```

File: iptv/kodi/kodi.m3u [hunk 6]
```diff
@@ -6,7 +6,7 @@ https://zdf-hls-15.akamaized.net/hls/live/2016498/de/veryhigh/master.m3u8
 #EXTINF:-1 tvg-name="3sat" tvg-id="3sat.de" group-title="Vollprogramm" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/3sat.png",3sat
 https://zdf-hls-18.akamaized.net/hls/live/2016501/dach/veryhigh/master.m3u8
 #EXTINF:-1 tvg-name="ARTE HD" tvg-id="ARTE.de" group-title="Vollprogramm" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/artehd.png",ARTE HD
-https://artelive-lh.akamaihd.net/i/artelive_de@393591/index_1_av-b.m3u8
+https://artelive-lh.akamaihd.net/i/artelive_de@393591/master.m3u8
```
Comment (jnk22) [inline iptv/kodi/kodi.m3u:9]:
```suggestion
https://artelive-lh.akamaihd.net/i/artelive_de@393591/index_1_av-p.m3u8
```

File: iptv/pipe/pipe_tv_main.m3u [hunk 7]
```diff
@@ -6,7 +6,7 @@ pipe://ffmpeg -loglevel fatal -i https://zdf-hls-15.akamaized.net/hls/live/20164
 #EXTINF:-1 tvg-name="3sat" tvg-id="3sat.de" group-title="IPTV-DE" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/3sat.png",3sat
 pipe://ffmpeg -loglevel fatal -i https://zdf-hls-18.akamaized.net/hls/live/2016501/dach/veryhigh/master.m3u8 -vcodec copy -acodec copy -metadata service_name=3sat -metadata service_provider=IPTV-DE -mpegts_service_type advanced_codec_digital_sdtv -f mpegts pipe:1
 #EXTINF:-1 tvg-name="ARTE HD" tvg-id="ARTE.de" group-title="IPTV-DE" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/artehd.png",ARTE HD
-pipe://ffmpeg -loglevel fatal -i https://artelive-lh.akamaihd.net/i/artelive_de@393591/index_1_av-b.m3u8 -vcodec copy -acodec copy -metadata service_name=ARTE\ HD -metadata service_provider=IPTV-DE -mpegts_service_type advanced_codec_digital_hdtv -f mpegts pipe:1
+pipe://ffmpeg -loglevel fatal -i https://artelive-lh.akamaihd.net/i/artelive_de@393591/master.m3u8 -vcodec copy -acodec copy -metadata service_name=ARTE\ HD -metadata service_provider=IPTV-DE -mpegts_service_type advanced_codec_digital_hdtv -f mpegts pipe:1
```
Comment (jnk22) [inline iptv/pipe/pipe_tv_main.m3u:9]:
```suggestion
pipe://ffmpeg -loglevel fatal -i https://artelive-lh.akamaihd.net/i/artelive_de@393591/index_1_av-p.m3u8 -vcodec copy -acodec copy -metadata service_name=ARTE\ HD -metadata service_provider=IPTV-DE -mpegts_service_type advanced_codec_digital_hdtv -f mpegts pipe:1
```

File: iptv/pipe/pipe_tv.m3u [hunk 8]
```diff
@@ -6,7 +6,7 @@ pipe://ffmpeg -loglevel fatal -i https://zdf-hls-15.akamaized.net/hls/live/20164
 #EXTINF:-1 tvg-name="3sat" tvg-id="3sat.de" group-title="IPTV-DE" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/3sat.png",3sat
 pipe://ffmpeg -loglevel fatal -i https://zdf-hls-18.akamaized.net/hls/live/2016501/dach/veryhigh/master.m3u8 -vcodec copy -acodec copy -metadata service_name=3sat -metadata service_provider=IPTV-DE -mpegts_service_type advanced_codec_digital_sdtv -f mpegts pipe:1
 #EXTINF:-1 tvg-name="ARTE HD" tvg-id="ARTE.de" group-title="IPTV-DE" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/artehd.png",ARTE HD
-pipe://ffmpeg -loglevel fatal -i https://artelive-lh.akamaihd.net/i/artelive_de@393591/index_1_av-b.m3u8 -vcodec copy -acodec copy -metadata service_name=ARTE\ HD -metadata service_provider=IPTV-DE -mpegts_service_type advanced_codec_digital_hdtv -f mpegts pipe:1
+pipe://ffmpeg -loglevel fatal -i https://artelive-lh.akamaihd.net/i/artelive_de@393591/master.m3u8 -vcodec copy -acodec copy -metadata service_name=ARTE\ HD -metadata service_provider=IPTV-DE -mpegts_service_type advanced_codec_digital_hdtv -f mpegts pipe:1
```
Comment (jnk22) [inline iptv/pipe/pipe_tv.m3u:9]:
```suggestion
pipe://ffmpeg -loglevel fatal -i https://artelive-lh.akamaihd.net/i/artelive_de@393591/index_1_av-p.m3u8 -vcodec copy -acodec copy -metadata service_name=ARTE\ HD -metadata service_provider=IPTV-DE -mpegts_service_type advanced_codec_digital_hdtv -f mpegts pipe:1
```

File: iptv/pipe/pipe.m3u [hunk 9]
```diff
@@ -6,7 +6,7 @@ pipe://ffmpeg -loglevel fatal -i https://zdf-hls-15.akamaized.net/hls/live/20164
 #EXTINF:-1 tvg-name="3sat" tvg-id="3sat.de" group-title="IPTV-DE" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/3sat.png",3sat
 pipe://ffmpeg -loglevel fatal -i https://zdf-hls-18.akamaized.net/hls/live/2016501/dach/veryhigh/master.m3u8 -vcodec copy -acodec copy -metadata service_name=3sat -metadata service_provider=IPTV-DE -mpegts_service_type advanced_codec_digital_sdtv -f mpegts pipe:1
 #EXTINF:-1 tvg-name="ARTE HD" tvg-id="ARTE.de" group-title="IPTV-DE" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/artehd.png",ARTE HD
-pipe://ffmpeg -loglevel fatal -i https://artelive-lh.akamaihd.net/i/artelive_de@393591/index_1_av-b.m3u8 -vcodec copy -acodec copy -metadata service_name=ARTE\ HD -metadata service_provider=IPTV-DE -mpegts_service_type advanced_codec_digital_hdtv -f mpegts pipe:1
+pipe://ffmpeg -loglevel fatal -i https://artelive-lh.akamaihd.net/i/artelive_de@393591/master.m3u8 -vcodec copy -acodec copy -metadata service_name=ARTE\ HD -metadata service_provider=IPTV-DE -mpegts_service_type advanced_codec_digital_hdtv -f mpegts pipe:1
```
Comment (jnk22) [inline iptv/pipe/pipe.m3u:9]:
```suggestion
pipe://ffmpeg -loglevel fatal -i https://artelive-lh.akamaihd.net/i/artelive_de@393591/index_1_av-p.m3u8 -vcodec copy -acodec copy -metadata service_name=ARTE\ HD -metadata service_provider=IPTV-DE -mpegts_service_type advanced_codec_digital_hdtv -f mpegts pipe:1
```

**Review by jnk22 — CHANGES_REQUESTED** [review]

**Review by jnk22 — APPROVED** [review]

**Pull request merged**